lib/usd/translators/CMakeLists.txt

```diff
@@ -46,6 +46,42 @@ target_sources(${TARGET_NAME}
         xformReader.cpp
 )
 
+set(headers
+    api.h
+    cameraReader.h
+    cameraWriter.h
+    fileTextureWriter.h
+    instancerWriter.h
+    jointWriter.h
+    lightReader.h
+    lightWriter.h
+    locatorWriter.h
+    materialReader.h
+    mayaReferenceReader.h
+    mayaReferenceUpdater.h
+    meshReader.h
+    meshWriter.h
+    meshWriter_Primvars.h
+    meshWriter_Skin.h
+    meshWriter_Subdiv.h
+    nurbsCurvesReader.h
+    nurbsCurveWriter.h
+    nurbsPatchReader.h
+    nurbsSurfaceWriter.h
+    particleWriter.h
+    scopeReader.h
+    skelReader.h
+    strokeWriter.h
+    xformReader.h
+)
```

Comment:
Many of these files do not exist. Here are the ones I'm currently seeing in `lib/usd/translators`:

```
api.h
cameraWriter.h
fileTextureWriter.h
instancerWriter.h
jointWriter.h
locatorWriter.h
mayaReferenceUpdater.h
meshWriter.h
nurbsCurveWriter.h
nurbsSurfaceWriter.h
particleWriter.h
strokeWriter.h
```

But more than that, they should all be private, so we probably should not be promoting them.

Reply:
Resolved in c6eecc56e9bf07aeaaf02f572b0ae1d588932226.